tardis/model/tests/test_config_model.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pandas as pd
+import tardis
+import os
+from tardis.io.config_reader import Configuration
+from tardis.model import Radial1DModel
+import pytest
+
+DATA_PATH = os.path.join(tardis.__path__[0],'model','tests','data')
+
+@pytest.fixture(scope='module')
+def config_hard_coded():
+    configpath = os.path.join(DATA_PATH, 'config_hard_coded.yml')
+    config = Configuration.from_yaml(configpath)
+    config_hard_coded = Radial1DModel.from_config(config)
+    return config_hard_coded 
+ 
+def test_config_input_abundance_storage(config_hard_coded):
```

Comment:
More descriptive function name and include docstring describing its purpose for these tests.